jupyterlab/tests/test_app.py

```diff
@@ -30,6 +30,13 @@
 HERE = osp.realpath(osp.dirname(__file__))
 
 
+try:
+    basestring
+    PY2 = True
+except NameError:
+    PY2 = False
+
+
```

Comment:
I'm not overly familiar with what the best idiom for doing this is, but I thought something like `sys.version_info < (3, 0)` is more idiomatic. What do you think?